colin-api/src/colin_api/resources/filing.py

```diff
@@ -214,7 +215,7 @@ def _add_filings(con, json_data: dict, filing_list: list, identifier: str) -> li
         for filing_type in filing_list:
             filing = Filing()
             filing.header = json_data['header']
-            filing.filing_date = filing.header['date']
+            filing.filing_date = convert_to_pacific_time(filing.header['date'])
```

Comment:
Which filings should be tested due to this change? Do you have screenshots?

Reply:
Generic to all filings

Reply:
Please let QA know in ticket test notes.